static/skin/taskbar.js

```diff
@@ -3,83 +3,86 @@ function htmlDecode(input) {
     return doc.documentElement.textContent;
 }
 
-(function ($) {
-    const root = $( `link[type='root']` ).attr("href");
-    const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
-
-    $( "#kiwixsearchbox" ).autocomplete({
-
-        source: `${root}/suggest?content=${bookName}`,
-        dataType: "json",
-        cache: false,
-
-        response: function( event, ui ) {
-              for(const item of ui.content) {
-                  item.label = htmlDecode(item.label);
-                  item.value = htmlDecode(item.value);
-                  if (item.path) item.path = htmlDecode(item.path);
-              }
-        },
-
-        select: function(event, ui) {
-            if (ui.item.kind === 'path') {
-                window.location.href = `${root}/${bookName}/${encodeURI(ui.item.path)}`;
-            } else {
-                $( "#kiwixsearchbox" ).val(ui.item.value);
-                $( "#kiwixsearchform" ).submit();
-            }
-        },
-    });
-
-    /* cybook hack */
-    if (navigator.userAgent.indexOf("bookeen/cybook") != -1) {
-        $("html").addClass("cybook");
-    }
-
-    if ($(window).width() < 520) {
-        var didScroll;
-        var lastScrollTop = 0;
-        var delta = 5;
-        // on scroll, let the interval function know the user has scrolled
-        $(window).scroll(function (event) {
-            didScroll = true;
+const jq = jQuery.noConflict();
+jq(document).ready(() => {
+    (function ($) {
```

Comment:
Your code is 
```
jq(document).ready(() => {
  (function ($) { ... })(jq);
})
```
But documentation propose 
```
jq(document).ready(function( $ ) {
   ...
});
```

The second seems simpler. What is the difference ?

Reply:
@mgautierfr the second code specifies to use the global variable $ which has the zim file variant of jquery associated with it, in the first type we are mentioning to use `jq` as a $.
P.S. - this code was not added by me and I just tried to keep it similar to older code.